pyupgrade/_main.py

```diff
@@ -483,29 +397,13 @@ def _fix_tokens(contents_text: str, min_version: Version) -> str:
     except tokenize.TokenError:
         return contents_text
     for i, token in reversed_enumerate(tokens):
-        if token.name == 'NUMBER':
-            tokens[i] = token._replace(src=_fix_long(_fix_octal(token.src)))
-        elif token.name == 'STRING':
+        if token.name == 'STRING':
             tokens[i] = _fix_ur_literals(tokens[i])
             if remove_u:
                 tokens[i] = _remove_u_prefix(tokens[i])
             tokens[i] = _fix_escape_sequences(tokens[i])
         elif token.src == '(':
             _fix_extraneous_parens(tokens, i)
-        elif token.src == 'format' and i > 0 and tokens[i - 1].src == '.':
```

Comment:
So walking through these to see if I have them right:
- Remove format for f-strings
- Remove `encode` in favor of byte literals
- Not sure on last one - something on newline tokens?

Reply:
Hey Ben, I ended up adding most of this back and putting the things that were questionable behind a fix flag, so they can be manually excluded